generated/balancer.tokenlist.json

```diff
@@ -14319,7 +14320,8 @@
             "address": "0x32dF62dc3aEd2cD6224193052Ce665DC18165841",
             "name": "RDNT-WETH",
             "symbol": "RDNT-WETH",
-            "decimals": 18
+            "decimals": 18,
+            "logoURI": "missing_large.png"
```

Comment:
RDNT/WETH still had missing image problems.

The other 2 found an ew token logo.